testplan/cli/commands/writers.py

```diff
@@ -124,7 +124,7 @@ def __init__(self, dir_name: str) -> None:
     def __call__(self, result: TestReport) -> TestReport:
         exporter = XMLExporter(xml_dir=self.dir_name)
         exporter.export(result)
-        self.logger.test_info(f"XML files written to {self.dir_name}")
+        self.logger.user_info(f"XML files written to {self.dir_name}")
```

Comment:
Noticed these are still f-strings instead of the lazy formatting. Can you please update while at it?